web/source/keymanweb.js

```diff
@@ -2272,15 +2269,16 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
      * Function     getKeyboardForControl
      * Scope        Public   
      * @param       {Element}    Pelem    Control element 
+     * @return      {string|null}         The independently-managed keyboard for the control.
      * Description  Returns the keyboard ID of the current independently-managed keyboard for this control.
      *              If it is currently following the global keyboard setting, returns null instead.
      */
     keymanweb['getKeyboardForControl'] = keymanweb.getKeyboardForControl = function(Pelem) {
       if(!keymanweb.isAttached(Pelem)) {
         console.error("KeymanWeb is not attached to element " + Pelem);
-        return;
+        return null;
       } else {
-        return Pelem._kmwAttachment.languageCode;
+        return Pelem._kmwAttachment.keyboard;  // Should we have a version for the language code, too?
```

Comment:
Good question to ask. ... yes we should. `getKeyboardForControl(pElem, { returnType: 'object' })` or some such. But that should be an issue, not added to this PR please.